src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/ElectionTermCollectorTests.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright <2019> Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
nit: can we change year to 2021 for all new files which need to make change to license header?

Reply:
sure